index.html

```diff
@@ -6,189 +6,297 @@
-            <a href="images/jasna17_projekt_dla_jaroszewskiego (3).png" data-lightbox="wiz"><img src="images/mini_jasna17_projekt_dla_jaroszewskiego (3).png" alt="Wizualizacja kuchni na ul. Jasnej 17 w Płocku dla Pana Jaroszewskiego, który stwierdził, że 300zł jest zbyt wygórowaną ceną" class="photo">
-            </a>
+        <h2>
+          Czym się zajmuję
+        </h2>
+        <div class="works-wrapper">
+          <h3>
+            Strony www
+            <small>(od nastarszych)</small>
+          </h3>
+          <ul class="works-list">
+            <li class="works-list-item">
+              <a href="http://www.przedtestem.edu.pl" target="_blank">
+                <img src="images/mini_przedtestem.png" alt="Strona główna portalu przedtestem.pl" class="photo">
+              </a>
+            </li>
+            <li class="works-list-item">
+              <a href="http://www.damianlewandowski.pl" target="_blank">
+                <img src="images/mini_damianlewandowski.png" alt="Strona główna damianlewandowski.pl" class="photo">
+              </a>
+            </li>
+            <li class="works-list-item">
+              <a href="http://www.babska-fanaberia.pl" target="_blank">
+                <img src="images/mini_babskafanaberia.jpg" alt="Strona główna babska-fanaberia.pl" class="photo">
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="works-wrapper">
+          <h3>
+            Wizualizacje
+          </h3>
+          <ul class="works-list">
+            <li class="works-list-item">
+              <a href="images/jasna17-projekt-dla-jaroszewskiego-3.png" data-lightbox="wiz">
+                <img src="images/mini-jasna17-projekt-dla-jaroszewskiego-3.png" alt="Wizualizacja kuchni na ul. Jasnej 17 w Płocku dla Pana Jaroszewskiego, który stwierdził, że 300zł jest zbyt wygórowaną ceną" class="photo">
+              </a>
+            </li>
+            <li class="works-list-item">
+              <a href="images/Lazienka-Gesia.jpg" data-lightbox="wiz">
+                <img src="images/mini-Lazienka-Gesia.png" alt="Wizualizacja łazienki w  domu Damiana Lewandowskiego" class="photo">
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="works-wrapper">
+          <h3>
+            Grafika 2D
+          </h3>
+          <ul class="works-list">
+            <li class="works-list-item">
+              <a href="images/krag_2d.png" data-lightbox="2d">
+                <img src="images/mini_krag_2d.png" alt="Tekstura, normal mapa, specular mapa dla modelu low-poly kosmicznego kręgu" class="photo">
+              </a>
+            </li>
+            <li class="works-list-item">
+              <a href="images/hustawka_2d.png" data-lightbox="2d">
+                <img src="images/mini_hustawka_2d.png" alt="Tekstura, normal mapa, specular mapa dla modelu low-poly hustawki do gry tworzonej przez Dariusza Furmana z Tarnowa" class="photo">
```

Comment:
It would be nice to change all images’ names to equivalents in English. I’m going to create a task about this. Don’t modify it in this PR.